alertwildfire_live_recording.py

```diff
@@ -94,15 +99,76 @@ def capture_and_record(googleServices, dbManager, outputDir, camera_name):
 
 
     image_base_name = pathlib.PurePath(imgPath).name
-    image_name_with_metadata = build_name_with_metadata(image_base_name,pull1)
+    #implement the ocr 
+    vals = OCR.pull_metadata("Axis", filename = imgPath ).split()
+    logging.warning('values read by OCR %s',vals)
+    if len(vals) == 0:
+        logging.warning('OCR Failed image recorded using motor information')
+        
+    metadata = {key:None for key in ["name", "date", "time","timeStamp","pan","tilt","zoom"]}
+    """ format
+    metadata = {
+                "name" : camera_name,
+                "date" : [elem for elem in vals if elem.count("/") == 2][0],
+                "time" : [elem for elem in vals if elem.count(":") == 2][0],
+                "timeStamp" : #unix timestamp
+                "pan" : float([elem for elem in vals if "X:" in elem][0][2:]),
+                "tilt" : float([elem for elem in vals if "Y:" in elem][0][2:]),
+                "zoom" : float([elem for elem in vals if "Z:" in elem][0][2:]),
+                 }"""
+    cases = {"pan"  :[''.join(elem) for elem in list(itertools.product(['X','x'],[':','.','_']))],
+             "tilt" :[''.join(elem) for elem in list(itertools.product(['Y','y','V','v'],[':','.','_']))],
+             "zoom" :[''.join(elem) for elem in list(itertools.product(['Z','z'],[':','.','_']))],
```

Comment:
Please document the reason for all these.  Examples may be helpful.

Reply:
name, date, and time will be useful to show in the UI, as a simple marker/notifications that inform the user about the basic detection information. I'm not sure if timeStamp will be required at this point--yes if above date/time is unusable directly and needs to be formatted via JavaScript; no if date/time above comes in perfectly usable format.

pan: crucial for informing the user where to look; also likely to be needed to draw an element on the map.
tilt: may not be necessary; not sure at this point
zoom: may not be necessary; not sure at this point

Reply:
Let me know if you want me to add those directly to the code rather than in this comment here.